src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -232,6 +231,22 @@ const moduleExports = {
 
 Note that this only applies to client-side builds, and requires the `SentryWebpackPlugin` to be enabled.
 
+### Widen the Upload Scope
+
+_(New in version 6.18.3)_
+
+If you find that there are in-app frames in your client-side stacktraces which aren't getting sourcemapped even when most others are, it is likely because they are from files in `static/chunks/` rather than `static/chunks/pages/`. By default, such files aren't uploaded because the majority of the files in `static/chunks/` only contain nextjs or third-party code, and are named in such a way that it's hard to tell relevant files (ones containing your code) apart from irrelevant ones.
```

Comment:
I'm worried about the words "tell" and "apart" being separated in a somewhat idiomatic phrase, so I've tried to simplify the sentence a bit. Also, should "nextjs" be "Next.js"? Not sure if there's a convention I'm unaware of here.

```suggestion
If you find that there are in-app frames in your client-side stack traces that aren't getting source-mapped even when most others are, it's likely because they are from files in `static/chunks/` rather than `static/chunks/pages/`. By default, such files aren't uploaded because the majority of the files in `static/chunks/` only contain nextjs or third-party code, and are named in such a way that it's hard to recognize relevant files (ones containing your code) among irrelevant ones.
```

Reply:
Went with "distinguish between," but otherwise used your edits.